gym_lowcostrobot/envs/lift_cube_env.py

```diff
@@ -238,14 +265,21 @@ def step(self, action):
         cube_pos = self.data.qpos[:3]
         cube_z = cube_pos[2]
         ee_id = self.model.body("moving_side").id
-        ee_pos = self.data.geom_xpos[ee_id]
+        ee_pos = self.data.xpos[ee_id]
```

Comment:
Can you elaborate on this? I find it very hard to understand what the difference.

Reply:
Yes! I discovered that the previous function does not actually update the end-effector position, and the new one actually updates